Olha_Lesiuk/HW_8/task_1.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+"""
+Implementing my own print function that should work on all operating systems.
+It directly displays anything written to it to the Console. 
+"""
+
+
+def print_to_stdout(*args, **kwargs):
+    for arg in args:
+        print(*args, file=sys.stdout)
+    else:
+        for key, value in kwargs.items():
+            print("%s == %s" % (key, value))
```

Comment:
you use print() function. It is not a solution. You should implement your own variant of this functionality

Reply:
Fixed